src/main/java/org/opensearch/ad/transport/GetAnomalyDetectorTransportAction.java

```diff
@@ -159,7 +160,7 @@ protected void getExecute(GetAnomalyDetectorRequest request, ActionListener<GetA
         String detectorID = request.getDetectorID();
         String typesStr = request.getTypeStr();
         String rawPath = request.getRawPath();
-        String entityValue = request.getEntityValue();
+        Entity entityValue = request.getEntityValue();
```

Comment:
nit - same thing here? Maybe rename to just `entity`?

Reply:
yes, changed